packages/@lightningjs/ui-components/src/components/ProgressBar/ProgressBar.js

```diff
@@ -100,7 +100,7 @@ export default class ProgressBar extends Base {
   }
 
   get announce() {
-    if (this._announce) {
+    if (this._announce != undefined) {
```

Comment:
can we use `!==` here?

Reply:
Tim had a reason for this operator : https://github.com/rdkcentral/Lightning-UI-Components/pull/208#discussion_r1188764673

Reply:
I don't think I follow - what are we specifically leaning on that requires the use of loose equality instead of strict? the comment explains why we're adding the comparison to undefined, not the use loose equality

Reply:
@THoj13 ☝️

Reply:
Ok, I will change it to strict operator then.

Reply:
@anthony9187 I used loose equality so that `this._announce` being `null` or `undefined` would have the same falsey result since:
```js
// this._announce != undefined
null != undefined // false
undefined != undefined // false

// this._announce !== undefined
null !== undefined // true
undefined !== undefined // false
```

I'm not sure if `null` would ever be the value of `this._announce` though, so perhaps doesn't need to be accounted for.

Reply:
Thanks for the explanation. I'm ok with using loose equality here then, let's just make sure we comment both of the spots(cause I'll definitely just go "hm let me fix this") and add some unit tests around it